src/main/scala/is/hail/expr/FunctionRegistry.scala

```diff
@@ -1540,6 +1559,12 @@ object FunctionRegistry {
     "pattern1" -> "Substring to replace.",
     "pattern2" -> "Replacement string.")
 
+  registerMethod("entropy", { (x: String) => entropy(x)
+  },
+    """
+    Computes Shannon's entropy of String.
```

Comment:
@cseed do you think this be defined on `Array[T]`? Or renamed something like `stringEntropy`?

Reply:
Either way, Laurent, we should give a definition here, happy to help.

Reply:
So I created the Scala generic as it really didn't make a difference. In my case, I need this to compute the entropy of a DNA seq (likely the usecase we'll encounter), so it makes sense to have it on String. Here, it is called as `"attgtg".entropy()`, so I don't think it needs to be renamed to `stringEntropy`. Happy to add it to `Array[T]` too if you think it's useful.